include/deal.II/base/mpi.h

```diff
@@ -20,6 +20,7 @@
 
 #include <deal.II/base/array_view.h>
 #include <deal.II/base/index_set.h>
+#include <deal.II/base/mpi.templates.h>
```

Comment:
I think the idea was to specifically not include this file here, but keep it separate. Do you remember why you needed it?

Reply:
In 147b71fd64585107f0780dbb0826814473a6d371 I moved the implementation of builtin_single_broadcast to `mpi.cc`, I needed that module for the family of functions `mpi_type_id`.